Mycelium/scripts/helpers/variable_loader.py

```
@@ -119,21 +119,21 @@ def fetch_variable(identifier: str, root: str | Path = '.') -> Dict[str, int]:
     # identifier may be a tag '#Variable' -> return all variable files
     if identifier.strip() == '#Variable' or identifier.strip().lower() == 'variable':
         res: Dict[str, int] = {}
-    for f in iter_md_files(rootp):
+        for f in iter_md_files(rootp):
```

Ruff (0.14.10)

Undefined name `iter_md_files`

(F821)